webapp/src/constructDataUrl.ts

```diff
@@ -1,7 +1,5 @@
-import path from 'path'
-
 export default function constructDataUrl(
     dataRef: string
 ): string {
-    return `http://localhost:42224/api/pkgs/${encodeURIComponent(path.dirname(dataRef))}/contents/${encodeURIComponent(path.basename(dataRef))}`
+    return `http://localhost:42224/api/data/${encodeURIComponent(dataRef)}`
```

Comment:
Why doesn't this route show up in `sdks/go/node/api/urlTemplates.go`?

Reply:
it's in the API spec and is implemented but the path wasn't put into that file. It probably should be added.

Reply:
It's meant to be the replacement to the `pkgs/{ref}` and `pkgs/{ref}/contents/{path}` endpoints.